IndomitableIbises/Bangazon-API-Sprint-1
# created the product_type model and view

## Link to Ticket
Closes #5

## Description of Proposed Changes
Added the Product_Type model, view and URL

## Steps to Test
1. git fetch --all
2. git checkout JCproducttypemodel
3. Delete existing database
4. python manage.py migrate
5. code .
6. look at the code and see if you have questions
7. python manage.py runserver
8. You should see three clickable urls on our page. Click on "http://localhost:8000/product_type/" and make sure you are able to add the name of the Product type.


## Impacted Areas in Application

* After this merge we should be able to uncomment the customer_id and product_id on the product so the foreign keys will be added
*Models, views, and urls are impacted by this code


## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
